helm-charts/common/ollama-service/values.yaml

```diff
@@ -0,0 +1,94 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for ollama-service.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+replicaCount: 1
+
+# This sets the container image more information can be found here: https://kubernetes.io/docs/concepts/containers/images/
+image:
+  repository: ollama/ollama
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "0.5.7"
+
+  # Uncomment the following line to set desired image pull policy if needed, as one of Always, IfNotPresent, Never.
+  pullPolicy: Always
```

Comment:
Please comment this line, and let the k8s default kicks in. see https://github.com/opea-project/GenAIInfra/blob/main/helm-charts/common/mm-embedding/values.yaml#L26